apps/daffio/src/app/docs/components/doc-viewer/doc-viewer.module.ts

```diff
@@ -26,7 +28,9 @@ import { DaffioDocsTableOfContentsModule } from '../table-of-contents/table-of-c
     DaffioApiListSectionComponent,
     DaffSidebarModule,
     DaffButtonModule,
+    ...DAFF_BREADCRUMB_COMPONENTS,
```

Comment:
I don't think it works like this? It's causing errors when I try to serve daffio via `npx ng s daffio -c dev-libs`

Reply:
hm I'm not sure that we can import components like this. It works with the regular build but not `libs`. This needs to be investigated further before we merge all the standalone stuff